modules/imap/handler_modules.php

```diff
@@ -757,13 +757,15 @@ public function process() {
             $imap = Hm_IMAP_List::connect($form['imap_server_id'], $cache);
             if (imap_authed($imap)) {
                 $quota_root = $imap->get_quota_root($folder ? $folder : 'INBOX');
-                $quota = $imap->get_quota($quota_root[0]['name']);
-                if ($quota) {
-                    $current = floatval($quota[0]['current']);
-                    $max = floatval($quota[0]['max']);
-                    if ($max > 0) {
-                        $this->out('quota', ceil(($current / $max) * 100));
-                        $this->out('quota_max', $max / 1024);
+                if ($quota_root) {
```

Comment:
his condition ensures that $quota_root is not null or false. However, there's still a potential issue if $quota_root is an empty array. The check if ($quota_root) will pass for an empty array, but accessing $quota_root[0]['name'] would still result in the "Trying to access array offset on null" warning. 
Can you check that $quota_root contains elements before accessing $quota_root[0]['name'] like:
`if ($quota_root && isset($quota_root[0]['name']))`.